2024-10-07 17:10:45,956 INFO    MainThread:38210 [wandb_setup.py:_flush():79] Current SDK version is 0.18.3
2024-10-07 17:10:45,956 INFO    MainThread:38210 [wandb_setup.py:_flush():79] Configure stats pid to 38210
2024-10-07 17:10:45,957 INFO    MainThread:38210 [wandb_setup.py:_flush():79] Loading settings from /Users/achula/.config/wandb/settings
2024-10-07 17:10:45,957 INFO    MainThread:38210 [wandb_setup.py:_flush():79] Loading settings from /Users/achula/achula/td8/Proyecto-PAP/wandb/settings
2024-10-07 17:10:45,957 INFO    MainThread:38210 [wandb_setup.py:_flush():79] Loading settings from environment variables: {}
2024-10-07 17:10:45,957 INFO    MainThread:38210 [wandb_setup.py:_flush():79] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-10-07 17:10:45,957 INFO    MainThread:38210 [wandb_setup.py:_flush():79] Applying login settings: {}
2024-10-07 17:10:45,957 INFO    MainThread:38210 [wandb_setup.py:_flush():79] Applying login settings: {}
2024-10-07 17:10:45,957 INFO    MainThread:38210 [wandb_init.py:_log_setup():532] Logging user logs to /Users/achula/achula/td8/Proyecto-PAP/wandb/run-20241007_171045-ft7cxzpx/logs/debug.log
2024-10-07 17:10:45,957 INFO    MainThread:38210 [wandb_init.py:_log_setup():533] Logging internal logs to /Users/achula/achula/td8/Proyecto-PAP/wandb/run-20241007_171045-ft7cxzpx/logs/debug-internal.log
2024-10-07 17:10:45,958 INFO    MainThread:38210 [wandb_init.py:init():617] calling init triggers
2024-10-07 17:10:45,958 INFO    MainThread:38210 [wandb_init.py:init():624] wandb.init called with sweep_config: {}
config: {}
2024-10-07 17:10:45,958 INFO    MainThread:38210 [wandb_init.py:init():667] starting backend
2024-10-07 17:10:45,960 INFO    MainThread:38210 [wandb_init.py:init():671] sending inform_init request
2024-10-07 17:10:45,968 INFO    MainThread:38210 [backend.py:_multiprocessing_setup():104] multiprocessing start_methods=spawn,fork,forkserver, using: spawn
2024-10-07 17:10:45,968 INFO    MainThread:38210 [wandb_init.py:init():684] backend started and connected
2024-10-07 17:10:45,984 INFO    MainThread:38210 [wandb_run.py:_label_probe_notebook():1346] probe notebook
2024-10-07 17:10:45,986 INFO    MainThread:38210 [wandb_init.py:init():779] updated telemetry
2024-10-07 17:10:46,033 INFO    MainThread:38210 [wandb_init.py:init():812] communicating run to backend with 90.0 second timeout
2024-10-07 17:10:49,272 WARNING MainThread:38210 [wandb_init.py:init():1256] interrupted
Traceback (most recent call last):
  File "/Users/achula/miniconda3/envs/td8/lib/python3.8/site-packages/wandb/sdk/wandb_init.py", line 1252, in init
    return wi.init()
  File "/Users/achula/miniconda3/envs/td8/lib/python3.8/site-packages/wandb/sdk/wandb_init.py", line 815, in init
    result = run_init_handle.wait(
  File "/Users/achula/miniconda3/envs/td8/lib/python3.8/site-packages/wandb/sdk/lib/mailbox.py", line 283, in wait
    found, abandoned = self._slot._get_and_clear(timeout=wait_timeout)
  File "/Users/achula/miniconda3/envs/td8/lib/python3.8/site-packages/wandb/sdk/lib/mailbox.py", line 130, in _get_and_clear
    if self._wait(timeout=timeout):
  File "/Users/achula/miniconda3/envs/td8/lib/python3.8/site-packages/wandb/sdk/lib/mailbox.py", line 126, in _wait
    return self._event.wait(timeout=timeout)
  File "/Users/achula/miniconda3/envs/td8/lib/python3.8/threading.py", line 558, in wait
    signaled = self._cond.wait(timeout)
  File "/Users/achula/miniconda3/envs/td8/lib/python3.8/threading.py", line 306, in wait
    gotit = waiter.acquire(True, timeout)
KeyboardInterrupt
2024-10-07 17:11:35,948 INFO    MainThread:38210 [wandb_settings.py:_apply_login():1970] Applying login settings: {}
2024-10-07 17:11:35,955 INFO    MainThread:38210 [wandb_settings.py:_apply_login():1970] Applying login settings: {}
2024-10-07 17:11:35,960 INFO    MainThread:38210 [wandb_setup.py:_flush():79] Current SDK version is 0.18.3
2024-10-07 17:11:35,961 INFO    MainThread:38210 [wandb_setup.py:_flush():79] Configure stats pid to 38210
2024-10-07 17:11:35,961 INFO    MainThread:38210 [wandb_setup.py:_flush():79] Loading settings from /Users/achula/.config/wandb/settings
2024-10-07 17:11:35,961 INFO    MainThread:38210 [wandb_setup.py:_flush():79] Loading settings from /Users/achula/achula/td8/Proyecto-PAP/wandb/settings
2024-10-07 17:11:35,961 INFO    MainThread:38210 [wandb_setup.py:_flush():79] Loading settings from environment variables: {}
2024-10-07 17:11:35,961 INFO    MainThread:38210 [wandb_setup.py:_flush():79] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-10-07 17:11:35,961 INFO    MainThread:38210 [wandb_setup.py:_flush():79] Applying login settings: {}
2024-10-07 17:11:35,961 INFO    MainThread:38210 [wandb_setup.py:_flush():79] Applying login settings: {}
2024-10-07 17:11:35,962 INFO    MainThread:38210 [wandb_init.py:_log_setup():532] Logging user logs to /Users/achula/achula/td8/Proyecto-PAP/wandb/run-20241007_171135-t4gobpuj/logs/debug.log
2024-10-07 17:11:35,962 INFO    MainThread:38210 [wandb_init.py:_log_setup():533] Logging internal logs to /Users/achula/achula/td8/Proyecto-PAP/wandb/run-20241007_171135-t4gobpuj/logs/debug-internal.log
2024-10-07 17:11:35,962 INFO    MainThread:38210 [wandb_init.py:init():617] calling init triggers
2024-10-07 17:11:35,962 INFO    MainThread:38210 [wandb_init.py:init():624] wandb.init called with sweep_config: {}
config: {}
2024-10-07 17:11:35,962 INFO    MainThread:38210 [wandb_init.py:init():667] starting backend
2024-10-07 17:11:35,962 INFO    MainThread:38210 [wandb_init.py:init():671] sending inform_init request
2024-10-07 17:11:35,966 INFO    MainThread:38210 [backend.py:_multiprocessing_setup():104] multiprocessing start_methods=spawn,fork,forkserver, using: spawn
2024-10-07 17:11:35,966 INFO    MainThread:38210 [wandb_init.py:init():684] backend started and connected
2024-10-07 17:11:35,972 INFO    MainThread:38210 [wandb_run.py:_label_probe_notebook():1346] probe notebook
2024-10-07 17:11:35,973 INFO    MainThread:38210 [wandb_init.py:init():779] updated telemetry
2024-10-07 17:11:36,006 INFO    MainThread:38210 [wandb_init.py:init():812] communicating run to backend with 90.0 second timeout
2024-10-07 17:12:11,307 WARNING MainThread:38210 [wandb_init.py:init():1256] interrupted
Traceback (most recent call last):
  File "/Users/achula/miniconda3/envs/td8/lib/python3.8/site-packages/wandb/sdk/wandb_init.py", line 1252, in init
    return wi.init()
  File "/Users/achula/miniconda3/envs/td8/lib/python3.8/site-packages/wandb/sdk/wandb_init.py", line 815, in init
    result = run_init_handle.wait(
  File "/Users/achula/miniconda3/envs/td8/lib/python3.8/site-packages/wandb/sdk/lib/mailbox.py", line 283, in wait
    found, abandoned = self._slot._get_and_clear(timeout=wait_timeout)
  File "/Users/achula/miniconda3/envs/td8/lib/python3.8/site-packages/wandb/sdk/lib/mailbox.py", line 130, in _get_and_clear
    if self._wait(timeout=timeout):
  File "/Users/achula/miniconda3/envs/td8/lib/python3.8/site-packages/wandb/sdk/lib/mailbox.py", line 126, in _wait
    return self._event.wait(timeout=timeout)
  File "/Users/achula/miniconda3/envs/td8/lib/python3.8/threading.py", line 558, in wait
    signaled = self._cond.wait(timeout)
  File "/Users/achula/miniconda3/envs/td8/lib/python3.8/threading.py", line 306, in wait
    gotit = waiter.acquire(True, timeout)
KeyboardInterrupt
2024-10-07 17:29:30,699 INFO    MainThread:38210 [wandb_settings.py:_apply_login():1970] Applying login settings: {}
2024-10-07 17:29:30,709 INFO    MainThread:38210 [wandb_settings.py:_apply_login():1970] Applying login settings: {}
2024-10-07 17:29:30,727 INFO    MainThread:38210 [wandb_setup.py:_flush():79] Current SDK version is 0.18.3
2024-10-07 17:29:30,728 INFO    MainThread:38210 [wandb_setup.py:_flush():79] Configure stats pid to 38210
2024-10-07 17:29:30,728 INFO    MainThread:38210 [wandb_setup.py:_flush():79] Loading settings from /Users/achula/.config/wandb/settings
2024-10-07 17:29:30,728 INFO    MainThread:38210 [wandb_setup.py:_flush():79] Loading settings from /Users/achula/achula/td8/Proyecto-PAP/wandb/settings
2024-10-07 17:29:30,728 INFO    MainThread:38210 [wandb_setup.py:_flush():79] Loading settings from environment variables: {}
2024-10-07 17:29:30,728 INFO    MainThread:38210 [wandb_setup.py:_flush():79] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-10-07 17:29:30,728 INFO    MainThread:38210 [wandb_setup.py:_flush():79] Applying login settings: {}
2024-10-07 17:29:30,728 INFO    MainThread:38210 [wandb_setup.py:_flush():79] Applying login settings: {}
2024-10-07 17:29:30,728 INFO    MainThread:38210 [wandb_init.py:_log_setup():532] Logging user logs to /Users/achula/achula/td8/Proyecto-PAP/wandb/run-20241007_172930-ax7tschf/logs/debug.log
2024-10-07 17:29:30,728 INFO    MainThread:38210 [wandb_init.py:_log_setup():533] Logging internal logs to /Users/achula/achula/td8/Proyecto-PAP/wandb/run-20241007_172930-ax7tschf/logs/debug-internal.log
2024-10-07 17:29:30,728 INFO    MainThread:38210 [wandb_init.py:init():617] calling init triggers
2024-10-07 17:29:30,728 INFO    MainThread:38210 [wandb_init.py:init():624] wandb.init called with sweep_config: {}
config: {}
2024-10-07 17:29:30,729 INFO    MainThread:38210 [wandb_init.py:init():667] starting backend
2024-10-07 17:29:30,729 INFO    MainThread:38210 [wandb_init.py:init():671] sending inform_init request
2024-10-07 17:29:30,732 INFO    MainThread:38210 [backend.py:_multiprocessing_setup():104] multiprocessing start_methods=spawn,fork,forkserver, using: spawn
2024-10-07 17:29:30,733 INFO    MainThread:38210 [wandb_init.py:init():684] backend started and connected
2024-10-07 17:29:30,749 INFO    MainThread:38210 [wandb_run.py:_label_probe_notebook():1346] probe notebook
2024-10-07 17:29:30,751 INFO    MainThread:38210 [wandb_init.py:init():779] updated telemetry
2024-10-07 17:29:30,811 INFO    MainThread:38210 [wandb_init.py:init():812] communicating run to backend with 90.0 second timeout
2024-10-07 17:31:01,165 ERROR   MainThread:38210 [wandb_init.py:init():838] encountered error: Run initialization has timed out after 90.0 sec. 
Please refer to the documentation for additional information: https://docs.wandb.ai/guides/track/tracking-faq#initstarterror-error-communicating-with-wandb-process-
2024-10-07 17:31:01,169 ERROR   MainThread:38210 [wandb_init.py:init():1262] error in wandb.init()
Traceback (most recent call last):
  File "/Users/achula/miniconda3/envs/td8/lib/python3.8/site-packages/wandb/sdk/wandb_init.py", line 1252, in init
    return wi.init()
  File "/Users/achula/miniconda3/envs/td8/lib/python3.8/site-packages/wandb/sdk/wandb_init.py", line 844, in init
    raise error
wandb.errors.errors.CommError: Run initialization has timed out after 90.0 sec. 
Please refer to the documentation for additional information: https://docs.wandb.ai/guides/track/tracking-faq#initstarterror-error-communicating-with-wandb-process-
2024-10-07 17:38:35,610 INFO    MainThread:38210 [wandb_settings.py:_apply_login():1970] Applying login settings: {}
